apps/stable_diffusion/web/ui/stablelm_ui.py

```diff
@@ -114,14 +121,20 @@ def chat(curr_system_message, history, model, device, precision, cli=True):
 
     if model_name in [
         "vicuna",
+        "vicuna4",
         "vicuna1p3",
         "codegen",
         "llama2_7b",
         "llama2_70b",
     ]:
-        from apps.language_models.scripts.vicuna import (
-            UnshardedVicuna,
-        )
+        if model_name == "vicuna4":
+            from apps.language_models.scripts.vicuna_4_shards import (
+                ShardedVicuna as UnshardedVicuna,
```

Comment:
nice

Reply:
but please change all mentions of unsharded vicuna to "as Vicuna". This will make it less confusing for anyone to read later